src/Console/GenerateCommandBuilder.php

```diff
@@ -0,0 +1,180 @@
+
+                    $iconFileList = $this->getDirectoryFileList($this->getSvgSourcePath() . DIRECTORY_SEPARATOR . $iconSetName, $iconSetConfig);
+                    $this->updateSvgs($iconSetConfig, $iconFileList);
+                    $output->writeln("Completed processing for '{$iconSetName}' svgs.");
+                }
+
+                $output->writeln("Cleaning up the build directory...");
+                $this->deleteDirectory(static::getSvgTempPath());
+                $output->writeln("Done!");
+
+                return Command::SUCCESS;
+            })
+            ->run();
+    }
+
+    private function getSvgSourcePath(): string
+    {
+        if ($this->npmPackageName !== null) {
+            return $this->baseDirectory . DIRECTORY_SEPARATOR .
+                    'node_modules' . DIRECTORY_SEPARATOR .
+                    $this->npmPackageName . DIRECTORY_SEPARATOR .
+                    ltrim($this->sourceDirectory, DIRECTORY_SEPARATOR);
+        }
+
+        return $this->baseDirectory . DIRECTORY_SEPARATOR . ltrim($this->sourceDirectory, DIRECTORY_SEPARATOR);
+    }
+
+    private function getSvgTempPath(): string
+    {
+        return $this->baseDirectory . DIRECTORY_SEPARATOR . 'build';
+    }
+
+    private function getSvgDestinationPath(): string
+    {
+        return $this->baseDirectory . DIRECTORY_SEPARATOR . 'resources' . DIRECTORY_SEPARATOR . 'svg';
+    }
+
+    private function ensureDirExists($dirPath)
+    {
+        if (!is_dir($dirPath)) {
+            if (!mkdir($dirPath) && !is_dir($dirPath)) {
+                throw new \RuntimeException(sprintf('Directory "%s" was not created', $dirPath));
+            }
+        }
+    }
+
+    /**
+     * @param string $dirPath
+     *
+     * @return array<array<string>>
+     */
+    private function getDirectoryFileList(string $dirPath, IconSetConfig $iconSetConfig): array
+    {
+        return array_diff(scandir($dirPath), ['..', '.']);
+    }
+
+    public function updateSvgs(IconSetConfig $iconSet, array $iconFileList): void
+    {
+
+        $iconSetName = $iconSet->name;
+        $iconSetTmpDir = $this->getSvgTempPath() . DIRECTORY_SEPARATOR . $iconSetName;
+
+        foreach ($iconFileList as $iconFile) {
+            // Set path variables...
+            $sourceFile = $iconSet->getSourceFilePath($iconFile);
+            $tempFile = $iconSet->getTempFilePath($iconFile);
+            $finalFile = $iconSet->getDestinationFilePath($iconFile, $this->useSingleIconSet);
+
+            // Copy file to temp...
+            copy($sourceFile, $tempFile);
+
+            // Apply user transformations if they provide them...
+            if ($this->svgNormalizationClosure !== null) {
+                $normalizeSvgClosure = $this->svgNormalizationClosure;
+                $normalizeSvgClosure($tempFile, $iconSet);
+            }
+
+            // Copy to final destination...
+            copy($tempFile, $finalFile);
```

Comment:
This appears to fail if the destination directory doesn't exist. 👍🏻 Probably worth having something like:
```suggestion
            mkdir(dirname($finalFile), 0777, true);
            copy($tempFile, $finalFile);
```

Reply:
Should be good now - used the `ensureDirExists` helper method.